statsrelay/src/config.rs

```diff
@@ -173,13 +176,34 @@ fn default_max_batch_size() -> usize {
 fn default_max_in_flight() -> usize {
     16
 }
+fn default_trim_on_disconnect() -> bool { false }
```

Comment:
Do `default_batch_creation_timeout_ms` and `default_max_batch_size` still seem reasonable for the prom remote write client? I originally set the batch creation timeout to 500ms to match the default statsd client config.

Reply:
I think 50ms is also reasonable for prom, the default max batch should definitely be lower as 10k would be >1MB on the wire. We probably want to aim closer to 100Kib. Let's go with 500 as the default batch.